app/bundles/ReportBundle/Translations/en_US/messages.ini

```diff
@@ -23,6 +23,11 @@ mautic.report.field.lead.title="Contact title"
 mautic.report.field.lead.zipcode = "Contact zip code"
 mautic.report.field.modified_by_user="Last modified by"
 mautic.report.field.publish_down="Publish down on"
+mautic.report.field.utm_campaign="UTM Campaign"
+mautic.report.field.utm_content="UTM Content"
+mautic.report.field.utm_medium="UTM Medium"
+mautic.report.field.utm_source="UTM source"
+mautic.report.field.utm_term="UTM Term"
```

Comment:
Can you use these translation strings from the LeadBundle instead?
```
mautic.lead.report.utm.campaign="UTM campaign"
mautic.lead.report.utm.content="UTM content"
mautic.lead.report.utm.medium="UTM medium"
mautic.lead.report.utm.source="UTM source"
mautic.lead.report.utm.term="UTM term"
```
I understand that this way there is dependency of the ReportBundle on the LeadBundle but this dependency exists already in so many places that the ReportBundle cannot work without LeadBundle anyway. I think we should use what's already there and save some time for our translators.